easybuild/easyconfigs/d/DRAGMAP/DRAGMAP-1.3.0-foss-2021b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'DRAGMAP'
+version = '1.3.0'
+
+homepage = 'https://github.com/Illumina/DRAGMAP'
+description = 'Dragmap is the Dragen mapper/aligner Open Source Software.'
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/Illumina/DRAGMAP/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['baf546c2f53b13e2bce4c937201e38459f3ce2f1069722ca670d8726a4bae224']
+
+skipsteps = ['configure']
+prebuildopts = 'sed -i "s|-msse4.2 -mavx2|$CXXFLAGS|g" config.mk &&'
+buildopts = 'CC="$CC" CXX="$CXX"'
+preinstallopts = 'mkdir %(installdir)s/bin &&'
+installopts = 'DESTDIR=%(installdir)s'
+
+dependencies = {
```

Comment:
@ItIsI-Orient for consistency with the other easyconfigs, please move the dependencies up to below the checksums?